src/components/App/App.js

```diff
@@ -1,58 +1,55 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import "./App.css";
 
-import SignIn from '../SignIn/SignIn'
-import AreasContainer from '../AreasContainer/AreasContainer'
-import ListingsContainer from '../ListingsContainer/ListingsContainer'
-import Favorites from '../Favorites/Favorites'
-import Header from '../Header/Header'
+import SignIn from "../SignIn/SignIn";
```

Comment:
I see you're on team semi-colon. A pair of star-crossed coders we are, alas!

Reply:
I'm actually team NO semi-colon since Mod 1 when @wvmitchell told me they do nothing but Prettier adds them in when I save